Assignment02/main.js

```diff
@@ -0,0 +1,195 @@
+            errorDiv.classList.add("errorBox"); //adding errorBox class to errorDiv
+            errorDiv.classList.add("visible"); //adding visible class to errorDiv
+        }
+    });
+    
+    mealForm.addEventListener("change", (e) => { //added event listener for change event on meal selection
+        makeHidden(mealOptionsArray); //setting meal choices for each respective meal selection to hidden
+        mealOptionsHolder.setAttribute("class", "visible"); //setting meal options div to visible
+        switch(mealForm.value){ //switch statement for mealForm value
+            case "breakfast": //if case is breakfast
+                        breakfast.setAttribute("class", "visible"); //setting breakfast options to visible
+                        break; //break out of switch
+            case "lunch": //if case is lunch
+                        lunch.setAttribute("class", "visible"); //setting lunch options to visible
+                        break; //break out of switch
+            case "dinner": //if case is dinner
+                        dinner.setAttribute("class", "visible"); //setting dinner options to visible
+                        break; //break out of switch
+            default: // default case for the options
+                        mealOptionsHolder.setAttribute("class", "hidden"); //div is set to hidden
+                        break; //break out of switch
+        }   
+    });
+
+    firstName.addEventListener("blur", () => { //on blur event of firstName input
+        let formatString = firstName.value; //setting value of firstName user entry to variable
+        let stringLength = formatString.length; //setting length to variable to use for slice method end index
+        let firstLetter = formatString[0].toUpperCase(); //capitalizing first letter of input and setting to variable
+        let restName = formatString.slice(1, stringLength); //slicing the rest of the userEntry after the first letter
+        formatString = firstLetter + restName.toLowerCase(); //concatenating the uppercase first letter and the rest of the entry
+        firstName.value = formatString; //setting form input value to formatted string
+    });
+    
+    lastName.addEventListener("blur", () => { //on blur event of lastName input
+        let formatString = lastName.value; //setting value of firstName user entry to variable
+        let stringLength = formatString.length; //setting length to variable to use for slice method end index
+        let firstLetter = formatString[0].toUpperCase(); //capitalizing first letter of input and setting to variable
+        let restName = formatString.slice(1, stringLength); //slicing the rest of the userEntry after the first letter
+        formatString = firstLetter + restName.toLowerCase(); //concatenating the uppercase first letter and the rest of the entry
+        lastName.value = formatString; //setting form input value to formatted string
```

Comment:
This process is *very* similar whether its first name or last name we're working with....function opportunity?